coral/src/app/features/configuration/clusters/components/ClusterConnectHelpModal.tsx

```diff
@@ -0,0 +1,227 @@
+import {
+  Box,
+  Button,
+  Icon,
+  PositionerPlacement,
+  Spacing,
+  Tooltip,
+  Typography,
+} from "@aivenio/aquarium";
+import duplicate from "@aivenio/aquarium/icons/duplicate";
+import link from "@aivenio/aquarium/icons/link";
+import { FormEvent, useEffect, useRef, useState } from "react";
+import { Modal } from "src/app/components/Modal";
+import { ClusterDetails } from "src/domain/cluster";
+
+const copyToClipboard = async (text: string): Promise<void> => {
+  try {
+    await navigator.clipboard.writeText(text);
+  } catch (error) {
+    console.error("Failed to copy to clipboard:", error);
+  }
+};
+
+const ClipBoard = ({
```

Comment:
Shouldn't this be a separate component?

Reply:
Yes, probably, this is definitely generic enough that it could stand on its own ^^

Reply:
I think this also addresses a11y issues [16c3153](https://github.com/Aiven-Open/klaw/pull/2315/commits/16c31536a476ebf4a24ed04e178fff4b693193d7)

https://github.com/Aiven-Open/klaw/assets/20607294/a91666c1-2569-4878-957c-42f3fe2d0105

Reply:
Also slightly changes the text for assertions:m [cc07fc3](https://github.com/Aiven-Open/klaw/pull/2315/commits/cc07fc3389f7f6964acc20b7896299251d2dfe9b)

Reply:
Nice - but the accessible description does nothing. The button label is not read out again when the text changes, so this does nothing. But it's a small thing, so I think we can merge that already and I can take a look at some point - I think I've an idea how to make that work, just needs to work in a nice way :D

And with the description it should be already more clear what is happening when the button is clicked.

Reply:
Ah and now I also noticed - the columnheader and aria-label for the row with the button lead to this being read: 

<img width="935" alt="Screenshot 2024-02-26 at 18 10 06" src="https://github.com/Aiven-Open/klaw/assets/943800/b2b7ab9e-5bbc-4d5a-b572-9fba8c3cbd8c">

I think we can remove the headerName and maybe update aria-label to: `Help for connecting cluster ${clusterName} to Klaw` Or `show help for connecting cluster ${clusterName} to Klaw`

Reply:
> Nice - but the accessible description does nothing. The button label is not read out again when the text changes, 

The video shows that it is?

<img width="550" alt="Screenshot 2024-02-26 at 14 06 06" src="https://github.com/Aiven-Open/klaw/assets/20607294/ca3ad886-fd47-48ea-b78d-ac495734b9de">

>  think we can remove the headerName and maybe update aria-label to: Help for connecting cluster ${clusterName} to Klaw Or show help for connecting cluster ${clusterName} to Klaw

Ah yes, sure!

Reply:
Ah damn, it shows with tab navigation, but not VO navigation. adding `aria-live="polite"` to the div works though ^^

Reply:
Yes, I also thought a `aria-live` should do the trick :D but I wasn't sure which one and so on. Works like a charm ✨